standalone_self_upgrade.cf.in

```diff
@@ -909,6 +956,14 @@ body package_module apt_get
       query_updates_ifelapsed => "30";
 }
 
+body package_module msiexec
+# @brief msiexec package module default settings
+{
+      query_installed_ifelapsed => "10";
```

Comment:
I would set this to 0 I think

Reply:
Why 0 for 3.12.x and not for 3.15.x and master?

Reply:
https://tracker.mender.io/browse/CFE-3504

Reply:
https://tracker.mender.io/browse/CFE-3613